session/persistence.go

```diff
@@ -29,6 +29,9 @@ type Persister interface {
 
 	// Delete removes a session from the store
 	DeleteSession(ctx context.Context, sid uuid.UUID) error
+
+	// Delete removes a session the store
```

Comment:
This should describe what the function is doing, so more something like:

```suggestion
	// DeleteSessionsFor removes all active session from the store for the given identity.
```